src/app/service/AFK/PublishAFKMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { MessageEmbedOptions } from 'discord.js';
+
+export const PublishAFKMessage = (username: string, content: string, guildId: string, channelId: string, id: string) : MessageEmbedOptions => {
+	return {
+		color: 1998388,
+		title: `${username} mentioned you!`,
+		timestamp: new Date().getTime(),
+		description: content,
+		url: `https://discord.com/channels/${guildId}/${channelId}/${id}`,
+		footer: {
+			text: 'Brought to you by DEGEN',
+		},
+	};
+};
```

Comment:
I'm wondering if this is necessary. Discord already has a feature for viewing mentions across servers or a specific server. If I was AFK, I would be annoyed by DMs from DEGEN, in addition to possible notifications from Discord.

Reply:
Thanks for all the suggestions duke! I committed all your suggestions and get to the others too.

For the PublishAFKMessage... all I can say is that I wrote the feature as requested in the issue by [Zrrow](https://github.com/BanklessDAO/discord-bots/issues/145).

Reply:
Very interesting, let's get a demo setup for next week's standup